content/zh/docs/community/developers.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "开发者指南"
+description: "开发Helm的环境设置说明。"
+weight: 1
+---
+
+该指南说明如何为开发Helm配置环境。
+
+## 先决条件
+
+- Go最新版本
+- Kubernetes集群及kubectl（可选）
+- Git
+
+## 构建 Helm
+
+我们使用Make构建程序。最简单的开始方式是：
+
+```console
+$ make
+```
+
+注意： 如果不是从`$GOPATH/src/helm.sh/helm`路径执行就会失败。目录`helm.sh`不应该是软链接，否则
+`build` 找不到相关的包。
+
+有必要的话要先安装依赖，重新构建 `vendor/` 目录，并验证配置。然后会编译 `helm` 并将其放到`bin/helm`。
+
+在本地执行Helm，可以执行 `bin/helm`。
+
+- Helm运行在 macOS 和大多数发行版上，包括 Alpine。
+
+## 执行测试
+
+要运行所有测试（除了`vendor/`），执行 `make test`。作为先决条件，需要安装
+[golangci-lint](https://golangci-lint.run)。
+
+## 贡献指南
+
+我们欢迎你的贡献。 该项目已经设置了一些指南，为了保证 (a) 代码的高质量，(b) 保持项目一致，
+(c) 贡献遵循开源法律要求。我们的目的不是为贡献者增加负担，但是要构建优雅和高质量的开源代码，
+这样我们的用户才能从中受益。
+
+确保你已经阅读并理解主要贡献指南：
+
+https://github.com/helm/helm/blob/master/CONTRIBUTING.md
+
+### 代码结构
+
+Helm项目的代码组织如下：
+
+- 独立的程序位于 `cmd/`。`cmd/` 中的代码不是为库复用设计的。
+- 共享的库放在 `pkg/`。
+- `scripts/` 目录包含很多实用程序脚本。大多数用于CI/CD流水线。
+
+Go依赖管理在不断变化，而且在Helm生命周期中很可能发生变化。我们建议开发者 _不要_ 手动管理依赖。
+而是建议依靠项目的 `Makefile` 来处理。使用Helm 3时，建议使用Go 1.13及更新版本。
+
+### 编写文档
+
+从Helm 3开始，文档已经移动到了它自己的仓库中。当编制新特性时，请编写随附文档并提交到
+[helm-www](https://github.com/helm/helm-www) 仓库。
+
+### Git 约定
+
+我们使用Git作为版本控制系统。 `master` 分支是当前开发候选分支。发布版本会打tag。 如果你正在为Helm v2打补丁，
+ `dev-v2` 分支是从Helm 2发布版本开的基础分支。
```

Comment:
I will merge this PR, but I suggest you edit this section to remove the mention of `dev-v2`; we no longer patch or release Helm 2, and I've submitted a PR to remove this line from the English and Korean docs: https://github.com/helm/helm-www/pull/1049

Reply:
OK.